src/test/run-make/libtest-json/output.json

```diff
@@ -0,0 +1,10 @@
+{ "type": "suite", "event": "started", "test_count": "4" }
```

Comment:
I'm pretty sure serde_json would render this without any spaces between the fields:

```json
{"type":"suite","event":"started","test_count":"4"}
```

If we expect to use serde eventually, do we want to change this here so we don't need to adjust the tests later on?

Reply:
I don't think we should refrain from changing tests.
The format is bound to change, and so does the tests.

Ultimately I would want to ignore formatting altogether.
Tried using"jq", but the CI server doesn't have it installed.
I have an alternative solution in mind, but I don't think we should worry about that as of now.

Reply:
Shit, I just realized I've added the wrong file to git.